packages/mdc-textfield/adapter.js

```diff
@@ -28,7 +28,10 @@ import MDCTextFieldLabelFoundation from './label/foundation';
  *   value: string,
  *   disabled: boolean,
  *   badInput: boolean,
- *   checkValidity: (function(): boolean)
+ *   validity: {
```

Comment:
This will need a BREAKING CHANGE comment when you go to merge, so don't forget!

Reply:
Ack!